lib/rubocop/cli.rb

```diff
@@ -77,6 +77,8 @@ def handle_exiting_options
 
       puts RuboCop::Version.version(false) if @options[:version]
       puts RuboCop::Version.version(true) if @options[:verbose_version]
+
+      configuration
```

Comment:
If you just inline the init code here at the other place you can simply use `@cfg` as it was done before. This seems like a reasonable solution to me.